src/app/frontend/deploy/deployfromsettings_controller.js

```diff
@@ -12,12 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import showNamespaceDialog from './createnamespace_dialog';
 import DeployLabel from './deploylabel';
 import {stateName as replicasetliststate} from 'replicasetlist/replicasetlist_state';
 
 // Label keys for predefined labels
-export const APP_LABEL_KEY = 'app';
-export const VERSION_LABEL_KEY = 'version';
+const APP_LABEL_KEY = 'app';
+const VERSION_LABEL_KEY = 'version';
```

Comment:
Because these are ES6 constants, so compiler already knows that they are `@const` (this was needed for ES5). Also, the compiler knows that this is a string, because the const is initialized when declared. So compiler already knows `/** @const {string} */`.

On the other hand, if it was `let APP_LABEL_KEY` we would need type annotations to make sure the type always stays the same.

So this is, basically, a syntactic sugar that allows us to define constants easily.